### ISSUE TITLE
v1.4 colorscheme(?) error

### ISSUE THREAD

**Issue by gatesbydude** [Kruphixx/producthud#1]
![image](https://github.com/Kruphixx/producthud/assets/91558690/8b1a1c2b-0bd4-4b22-83e3-8738d853bdd7)
yeah well whatever this is, I am on GNU/Linux btw, and I just installed the newest version of the HUD.

**Comment by Kruphixx** [comment]
Yeah I recommend that you still download the very latest version and not the one from releases since usually it quickly becomes outdated. Fixed the issue around 2 days ago in https://github.com/Kruphixx/producthud/commit/c43b7a7dd938799109fd2b4b4fdb6fdcf45f535a
